### ISSUE TITLE
CMake OSX configuration

### ISSUE THREAD

**Issue by hadim** [lightvector/KataGo#690]
I am building conda packages for katago and the current builds requires the CMake config to be patched to work on osx.

Here is the patch: https://github.com/conda-forge/staged-recipes/pull/20777#discussion_r997638970

**Comment by ChinChangYang** [comment]
I can build KataGo without the change of `CMakeLists.txt` in my MacBook M1 Pro, so I suspect this is a Conda-specific issue.

Does Conda use its own compiler?

**Comment by hadim** [comment]
Yes conda setup things specifically to be work on any CI used by conda, so such things happens often. I am not asking you to modify your CMake file here if you don't want. I opened the ticket more for awareness about it. Maintaining a patch in the conda feedstock is totally fine on my side.
